py/desisim/scripts/quickquasars.py

```diff
@@ -67,8 +67,7 @@ def parse(options=None):
     parser.add_argument('--sigma_kms_fog',type=float,default=150, help="Adds a gaussian error to the quasar redshift that simulate the fingers of god effect")
 
     parser.add_argument('--gamma_kms_zfit',nargs='?',type=float,const=200,help="Adds a Lorentzian distributed shift to the quasar redshift, to simulate the redshift fitting step. E.g. --gamma_kms_zfit 200 will use a gamma parameter of 200 km/s . If a number is not specified the default is used.")
-
-
+    parser.add_argument('--shift_kms_los',nargs='?',type=float,const=100,help="Adds a shift to the quasar redshift written in the zbest sfile. If a number is not specified the default (100km/s) is used.")
```

Comment:
@londumas   there is no big difference other than sintaxis. 

'--shift_kms_los', nargs='?',type=float,const=100.  Allows to run with  --shift_kms_los with the "constant" value of a 100 without specifying the value. We only specify a number  if need something different. 

On the other hand we can do: 
'--shift_kms_los',type=float,default=0  so that we run with "--shift_kms_los 100"  

This is why I was saying if you find it confusing we could do whatever is less confusing.

Reply:
'--shift_kms_los',type=float,default=0 so that we run with "--shift_kms_los 100".
I would prefer that. It is a way more stable solution.
Thanks @alxogm.

Reply:
@londumas I think a more formal difference between "default" and "const" is that const only gets defined when the argument (--shift_kms_los) is present. "default" always define it, that is why in this case it would be best to use "default=0"